src/strategies/offer_maker/abstract/Direct.sol

```diff
@@ -18,6 +18,9 @@ import {IOfferLogic} from "mgv_src/strategies/interfaces/IOfferLogic.sol";
 
 ///@title `Direct` strats is an extension of MangroveOffer that allows contract's admin to manage offers on Mangrove.
 abstract contract Direct is MangroveOffer {
+  ///@notice `reserveId` is set in the constructor
+  event SetReserveId(address indexed reserveId);
```

Comment:
Consider adding base/quote event to HasIndexedBidsAndAsks
Consider adding mgv event somewhere so that we know which mgv a contract is attached to. We can derive it from seeder, but an event is easier to discover.